plugins/scaffolder-backend/package.json

```diff
@@ -36,6 +36,7 @@
     "git-url-parse": "^11.1.2",
     "globby": "^11.0.0",
     "helmet": "^4.0.0",
+    "jsonschema": "^1.2.6",
```

Comment:
json-schema is used in catalog model but it's not well maintained - the implementation anway. The test case I wrote passed validation (with no input) as it seems it doesn't treat the `required` field in the correct way. I've gone with this one but there doesn't really seem to be a standout that's compatibile with the json-schema standard.